### PR TITLE
Go Fix <unknown line number> lines

### PR DESCRIPTION
**Pull request by partouf**
Fixes unknown line number causing asm cut-offs from Go 1.10 output (issued raised in #901)

### CODE REVIEW

**Comment by partouf** [comment]
Added test

File: test/golang-tests.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+chai.use(chaiAsPromised);
+chai.should();
+
+const props = (key, deflt) => deflt;
+
+const ce = new CompilationEnvironment(props);
+const info = {
+    "exe": null,
+    "remote": true,
+    "lang": "pascal"
```
Comment (RabsRincon) [inline test/golang-tests.js]: You probably want to change this to go 👍

File: test/golang-tests.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2018, Patrick Quist
```
Comment (RabsRincon) [inline test/golang-tests.js]: I'm starting to use "201x, Compiler Explorer Authors" in case you want to do the same, although it's not required :)

**Comment by mattgodbolt** [comment]
Love it! Thanks for the tests, and the succinct fix :)

**Pull request merged**